atom/common/native_mate_converters/network_converter.cc

```diff
@@ -26,10 +26,9 @@ Converter<scoped_refptr<network::ResourceRequestBody>>::ToV8(
     auto post_data_dict = std::make_unique<base::DictionaryValue>();
     auto type = element.type();
     if (type == network::DataElement::TYPE_BYTES) {
-      auto vec = std::vector<char>(
+      auto bytes = std::make_unique<base::Value>(std::vector<char>(
           element.bytes(),
-          element.bytes() + static_cast<size_t>(element.length()));
-      auto bytes = std::make_unique<base::Value>(vec);
+          element.bytes() + static_cast<size_t>(element.length())));
```

Comment:
Why `static_cast<size_t>` ?

Reply:
excellent question, this was pre-existent to the changes i made so i left it in place 🤔